README.md

```diff
@@ -6,14 +6,14 @@ To deploy TensorFlow Serving with a model, start by deploying the charm. For a s
 will look something like this:
 
     juju deploy cs:~kubeflow-charmers/tf-serving \
-        --storage models=$NAME_OF_STORAGE_CLASS,, \
```

Comment:
The extra commas are actually necessary, see `juju add-storage --help` for this bit of explanatory text:

```
<storage-constraint> is a description of how Juju should provision storage
instances for the unit. The following three forms are accepted:

    <storage-pool>[,<count>][,<size>]
    <count>[,<size>]
    <size>
```

So it specifies the storage pool name, and gives empty values for count and size

Reply:
Okay, I'll remove that commit.